File: api/webapi/Dtos/KhachHang/UpdateKhachHangDto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace webapi.Dtos.KhachHang
{
    public class UpdateKhachHangDto
    {
        public string? TenKh { get; set; }

        public string? Email { get; set; }

        public string? Sdt { get; set; }

        public string? DiaChi { get; set; }

        public string? MaLoaiKh { get; set; }
    }
}

File: client/quan_ly_ban_hang/src/main/java/com/mycompany/Dtos/KhachHang/UpdateKhachHangDto.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */
package com.mycompany.Dtos.KhachHang;

/**
 *
 * @author ACER
 */
public class UpdateKhachHangDto {
    private String tenKh;
    private String email;
    private String sdt;
    private String diaChi;
    private String maLoaiKh;

    public String getTenKh() {
        return tenKh;
    }

    public void setTenKh(String tenKh) {
        this.tenKh = tenKh;
    }

    public String getEmail() {
        return email;
    }

    public void setEmail(String email) {
        this.email = email;
    }

    public String getSdt() {
        return sdt;
    }

    public void setSdt(String sdt) {
        this.sdt = sdt;
    }

    public String getDiaChi() {
        return diaChi;
    }

    public void setDiaChi(String diaChi) {
        this.diaChi = diaChi;
    }

    public String getMaLoaiKh() {
        return maLoaiKh;
    }

    public void setMaLoaiKh(String maLoaiKh) {
        this.maLoaiKh = maLoaiKh;
    }    
}
